pegasus/sites.v3/hourofcode.com/views/how-to/steps/steps_public_officials.haml

```diff
@@ -1 +1,151 @@
+:ruby
+  steps_educators_index = 9
 
+  steps_educators_heading = [
+    hoc_s(:hoc_how_to_step_label_invite_elected_official),
+    hoc_s(:hoc_how_to_step_label_register_event),
+    hoc_s(:hoc_how_to_step_label_choose_activities),
+    hoc_s(:hoc_how_to_step_label_plan_your_event),
+    hoc_s(:hoc_how_to_step_label_coordinate_media),
+    hoc_s(:hoc_how_to_step_label_confirm_details),
+    hoc_s(:hoc_how_to_step_label_get_coding),
+    hoc_s(:hoc_how_to_step_label_after_event),
+    hoc_s(:hoc_how_to_step_label_beyond_hoc)
+  ]
+
+  steps_educators_desc = [
+    hoc_s(:hoc_how_to_step_public_officials_invite_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_register_desc_01),
+    hoc_s(:hoc_how_to_step_choose_activities_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_plan_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_media_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_confirm_desc_01),
+    hoc_s(:hoc_how_to_step_public_officials_get_coding_desc_01),
+    hoc_s(:hoc_how_to_step_events_after_desc_01),
+    hoc_s(:hoc_how_to_step_beyond_advocacy_desc_01)
+  ]
+
+%section.steps
+  .wrapper
+
+    = view :"how-to/steps/side-nav/side_nav_public_officials"
+
+    %div.steps-wrapper
+      - steps_educators_index.times do |index|
+        .step{id: "step-0#{index + 1}", class:"#{'divider-accent' if index === 0 || index === 6 || index === 8}"}
+          .step-counter
+            %span
+              = index + 1
+          .step-content
+            %h3.heading-xl
+              = steps_educators_heading[index]
+            %p
+              = steps_educators_desc[index]
+
+            -# Step 1 additional content
+            - if index === 0
```

Comment:
I'm curious how much we're getting out of the loop, given there's custom code for every step? I know there's a little bit of shared code above but it's pretty straightforward

Reply:
I've used similar loops with all of the guides and find them easier to parse. Originally some of the steps didn't have additional content outside of the heading and paragraph, but the guides have been reworked a few times since then. 

Is there a benefit to not using a loop?

Reply:
I think I find them a little harder to parse so it's down to personal preference! If they're clearer for you, that's the direction we should go in :)